cmd/protoc-gen-crd/main.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Istio Authors
```

Comment:
Nit here and in other files. I believe we are going with just `// Copyright Istio Authors` and not entering the year, but not sure where we documented that. I see a smattering of years here.